device-types/Cisco/C8300-2N2S-4T2X.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Cisco
+model: C8300-2N2S-4T2X
+slug: cisco-c8300-2n2s-4t2x
+part_number: C8300-2N2S-4T2X
+u_height: 2
+is_full_depth: false
+airflow: front-to-rear
+is_powered: true
+comments: '[Cisco 8000 Series Routers Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/catalyst-8300-series-edge-platforms/datasheet-c78-744088.html)'
+console-ports:
+  - name: con0
+    type: rj-45
+power-ports:
+  - name: PSU0
+    type: iec-60320-c14
+    maximum_draw: 650
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 650
```

Comment:
These appear to be modular. Is that true? If so then please model them in `module-bays` and ensure the associated PSUs are modelled in the module-types folder

Reply:
I've added all the bays I could think of, and the PSUs we'd possibly use